contracts/Everlong.sol

```diff
@@ -1,11 +1,9 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity 0.8.20;
+pragma solidity ^0.8.20;
```

Comment:
What's the reasoning behind this change? We generally lock versions so that it's clear which compiler version we're using. Something that I've been thinking about is that we could/should loosen this up for interfaces and libraries so that if you want to bump the version of Everlong to, say, `0.8.22`, you could still use the Hyperdrive interfaces and libraries. That said, I still think we'll want to lock the version of this contract.